tgui/packages/tgui/interfaces/Vote.js

```diff
@@ -0,0 +1,229 @@
+import { useBackend } from "../backend";
+import {
+  Box,
+  Icon,
+  Flex,
+  Button,
+  Section,
+  Collapsible,
+} from "../components";
+import { Window } from "../layouts";
+import { logger } from "../logging";
+
+export const Vote = (props, context) => {
+  const { data } = useBackend(context);
+  const { mode, question, lower_admin } = data;
+  
+  return (
+    <Window
+      resizable
+      title={`Vote${
+        mode
+          ? `: ${
+            question
+              ? question.replace(/^\w/, c => c.toUpperCase())
+              : mode.replace(/^\w/, c => c.toUpperCase())
+          }`
+          : ""
+      }`}
+      width={400}
+      height={500} >
+      <Window.Content overflowY="scroll">
+        <Flex direction="column" height="100%">
+          {!!lower_admin && <AdminPanel />}
+          <ChoicesPanel />
+          <TimePanel />
+        </Flex>
+      </Window.Content>
+    </Window>
+  );
+};
+
+// Collapsible panel for admin actions.
+const AdminPanel = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { avm, avr, avmap, voting, upper_admin } = data;
+  return (
+    <Flex.Item>
+      <Section mb={1} title="Admin Options">
+        <Collapsible title="Start a Vote">
+          <Flex mt={2} justify="space-between">
+            <Flex.Item>
+              <Box mb={1}>
+                <Button
+                  disabled={!upper_admin || !avmap}
+                  onClick={() => act("map")} >
+                  Map
+                </Button>
+                {!!upper_admin && (
+                  <Button.Checkbox
+                    ml={1}
+                    color="red"
+                    checked={!avmap}
+                    onClick={() => act("toggle_map")} >
+                    Disable{!avmap ? "d" : ""}
```

Comment:
Move all the contents of the buttons to the content prop inside the button, goes for all the buttons ere

Reply:
No, I'm deprecating content in next tgui release.